repos/fdbt-site/src/pages/contact.tsx

```diff
@@ -36,11 +36,16 @@ const Contact = ({ supportEmail, supportPhone }: ContactProps): ReactElement =>
                     </p>
                     <h3 className="govuk-heading-s">Related services</h3>
                     <p className="govuk-body">
-                        If your query relates to the use of the Bus Open Data Service go&nbsp;
-                        <a href="https://publish.bus-data.dft.gov.uk/" aria-label="go to the bus open data service">
-                            here
+                        <a
+                            href="https://www.bus-data.dft.gov.uk/contact/"
+                            aria-label="go to the bus open data service"
+                            className="underline govuk-link"
+                        >
+                            https://www.bus-data.dft.gov.uk/contact/
```

Comment:
I'm not sure using the URL as the text is the right option (and for the other similar changed links in this PR) - the gov UK [Writing link text](https://www.gov.uk/guidance/content-design/links#writing-link-text) section says to make the link text descriptive

Reply:
this was done for disruptions and also in the main bods application as we didn't want to repeat the description

Reply:
What do you mean by "repeat the description"?

Reply:
avoiding repeating BODs so preventing something like this If your query relates to the use of the Bus Open Data Service go to the Bus Open Data Service.